textrepo-app/src/main/java/nl/knaw/huc/resources/task/DeleteDocumentResource.java

```diff
@@ -30,7 +30,7 @@ public DeleteDocumentResource(TaskBuilderFactory factory) {
   @ApiOperation(value = "Delete a document including its metadata, files, versions and contents. " +
       "Contents are only only deleted when not referenced by any other versions.")
```

Comment:
Ha, nu zie ik door de review zelf nog een foutje in de docs hier: "are only only deleted". :-)